builder-support/specs/pdns-recursor.spec

```diff
@@ -35,12 +35,8 @@ BuildRequires: libatomic
 %if 0%{?rhel} >= 7
 BuildRequires: protobuf-compiler
 BuildRequires: protobuf-devel
-
-%if 0%{?rhel} == 7
```

Comment:
Maybe
```
%if 0%{?rhel} >= 7
```
instead?

Reply:
The statement is already in a `%if 0%{?rhel} >= 7` condition (line 35)

Reply:
Oh, yes, sorry!